### PR TITLE
[BLOG] Add moving-scipy-to-meson

### PR DESCRIPTION
**Pull request by MarsBarLee**
This blog post is being resubmitted with the new workflow:
- Branching directly from the Quansight-website repository instead of a fork
- which allows me to authorize deployment of Vercel to build preview URLs
- making it easier to catch errors and make changes as necessary

The previous blog post PR, #361, is closed.

## Links to other Quansight blogs:

no labs links found
no llc links found

## Text styling

- [x] The blog is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [x] If there are headers, they use the proper header tags in order to do so (with only one level-one header).
- [x] All links describe where they link to (for example, check the [Quansight labs website](https://labs.quansight.org/)).
- [x] Any kind of styling that the author uses (for example, bold for emphasis) is consistent throughout the blog.

## Non-text contents

- [x] All content is represented as text (for example, images need alt text and videos need captions or descriptive transcripts).
- [x] If there are emojis, there are not more than three in a row.
- [x] Don't use [flashing gifs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [x] If it were to be read as plain text, the blog still makes sense and no information is missing.

### CODE REVIEW

File: apps/labs/posts/moving-scipy-to-meson.md
Comment (gabalafou) [inline apps/labs/posts/moving-scipy-to-meson.md:12]:
```suggestion
  alt: "Cropped diagram of SciPy's external build and runtime dependencies."
```

**Comment by MarsBarLee** [comment]
Hi @gabalafou, I’ve marked this PR as ready to review since the quansight-labs-live Vercel build is successful and all images are loading. Your re-deployment cleared up the SIGTERM error.

As you predicted, I cannot re-deploy Vercel myself. For future blog post PRs, if I get the SIGTERM error again, I’ll do the 2 commit method you suggested.

**Review by gabalafou — APPROVED** [review]
Thanks, this looks great!

Might I recommend a squash merge into the `develop` branch for a cleaner commit history?

Also, concerning the sigterm issue, feel free to ping me on Slack for a redeploy if I'm around. It's faster than the two-commit method, which is really just something I thought of off the top of my head to keep you from getting blocked (since not everyone has an account on Vercel since they're expensive and don't have a lot of safeguards to prevent someone from accidentally breaking the site).

**Comment by noatamir** [comment]
@MarsBarLee we are expecting a merge to main soon. If this (and maybe others?) blog is ready, I suggest you squash merge it to develop, so it will get publish in the next develop -> merge 🙇 💐 Todanke!

**Comment by MarsBarLee** [comment]
@gabalafou, @noatamir  I was able to squash merge to `develop`! The changes is reflected in the [develop commit history](https://github.com/Quansight/Quansight-website/commit/f465af6316c53e1d353e8f18e9d604d6f6c3b600).

There was a `protected branch` error. I was invited to the [quansight-website-dev and](https://github.com/orgs/Quansight/teams/quansight-website-dev) gained merge permission.

Should I close this PR, now what the changes are in `develop`?

**Comment by MarsBarLee** [comment]
When I squash-merge to develop last week, I was in the [quansight-web-dev](https://github.com/orgs/Quansight/teams/quansight-website-dev) admin team BUT not in the regular [Quansight Developers team](https://github.com/orgs/Quansight/teams/developers). Thus I had unusual permissions and Github didn't close this PR after the code was merged to develop.

Gabriel and I decided that I will leave quansight-web-dev team. I have requested permission to join the Quansight Developers team. Moving forward I will make these Labs Blogpost Migration PRs with Quansight Developers permissions.

**Pull request closed without merging**
